test/cypress/e2e/delegates.spec.js

```diff
@@ -76,6 +84,11 @@ describe('Delegates', () => {
     cy.get('@dg').find(ss.delegateProductivity).contains(/\d+ %/);
   });
 
+  /**
+   * Search functioning correctly
+   * @expect no delegates are shown if search for non-existing name
+   * @expect delegate is shown in 1 row when is found
+   */
```

Comment:
for this I dont understand what exactly means

Reply:
@Efefefef please ask @osvaldovega what is he missing here.